nextstrain/cli/runner/conda.py

```diff
@@ -524,6 +541,13 @@ def update() -> RunnerUpdateStatus:
     print(f"Updating Conda packages in {PREFIX}…")
     print(f"  - {update_spec}")
 
+    if osx_arm64_emulated():
+        requested_version = PackageSpec.parse(update_spec).version_spec or ""
```

Comment:
One issue that stands out here is that `version_spec` is not a bare version, e.g. it's often something like `==XYZ`. Comparing it with `>=` against a bare version won't work as this code expects.

Reply:
Oof, I should have done more testing...